### DISCUSSION TITLE
huma.Register vs Huma.Get

### DISCUSSION THREAD

**Discussion by joa23** [danielgtaylor/huma#400] [Q&A] [answered]
What is the difference between huma.Register vs Huma.Get/Post etc?

**Comment by danielgtaylor** [comment, marked as answer]
@joa23 `huma.Get` is just a convenience wrapper around `huma.Register`, which allows for more customization. See here: https://github.com/danielgtaylor/huma/blob/main/huma.go#L1449-L1478

It's easy to get started with `huma.Get`, then move to `huma.Register` if you can to customize anything from the operation ID, custom documentation, adding tags, customizing schema generation, adjusting default body read timeouts, etc.

**Reply by joa23** [reply]
Thanks.
